src/components/About.js

```diff
@@ -1,12 +1,18 @@
 import React from "react";
+import Links from "./Links"
+
+function About(props) {
+  let bioTag = undefined
+  if (props.bio !== undefined && props.bio !== "") {
```

Comment:
Since both `undefined` and `""` are falsey, you could do this:
```suggestion
  if (props.bio) {
```